apps/meteor/app/livechat/server/lib/Contacts.ts

```diff
@@ -136,9 +137,11 @@ export const Contacts = {
 		if (rooms?.length) {
 			for await (const room of rooms) {
 				const { _id: rid } = room;
-				(await Rooms.setFnameById(rid, name)) &&
-					(await LivechatInquiry.setNameByRoomId(rid, name)) &&
-					(await Subscriptions.updateDisplayNameByRoomId(rid, name));
+				if (await Rooms.setFnameById(rid, name)) {
```

Comment:
this has a potential to be a `Promise.all`  🤔 either would be different from original implementation.. so idk =P